hack/scripts/uuid_migration.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+if [[ $VERBOSE ]]; then
+  set -ex
+else
+  set -e
+fi
+
+OPS_PROJECTS=("default" "openshift" "openshift-infra")
+PROJECTS=(`oc get project | grep -v 'NAME' | cut -d' ' -f 1`)
+
+ES_PODS=$(oc get pod | grep 'logging-es' | grep 'Running' | grep -v '\-ops\-' | cut -d' ' -f 1)
+ES_POD=`echo $ES_PODS | cut -d' ' -f 1`
+
+CA=/etc/elasticsearch/keys/admin-ca
+KEY=/etc/elasticsearch/keys/admin-key
+CERT=/etc/elasticsearch/keys/admin-cert
+
+es_host=${ES_HOST:-logging-es}
+es_port=${ES_PORT:-9200}
+
+function create_alias() {
+
+    output=`oc exec $ES_POD -- curl -s --cacert $CA --key $KEY --cert $CERT -XPOST "https://$es_host:$es_port/_aliases" -d "{ \"actions\": [ { \"add\": { \"index\": \"${1}.*\", \"alias\": \"${1}.${2}.reference\"}} ] }"`
+
+    echo Migration for project $1: $output
+
+}
+
+for project in "${PROJECTS[@]}"; do
+
+  if [[ ! ( ${OPS_PROJECTS[@]} =~ $project ) ]]; then
+    uid=$(oc get project "$project" -o yaml | grep 'uid:' | cut -d' ' -f 4)
```

Comment:
thanks, for some reason i couldn't get that to work before... i must have been missing something